config/locales/links/en.yml

```diff
@@ -12,7 +12,7 @@ en:
       auth_app_fallback_html: ' or %{link}.'
       fallback_to_sms_html: Send me a text message with the code instead
       fallback_to_voice_html: If you can't get text message right now, you can get a security code via %{link}
-    privacy_policy: Privacy Policy
+    privacy_policy: Privacy & security
```

Comment:
We could - the ampersand was intended to save a little space on the mobile version of the footer. Say more about why you'd like the full word!